src/swap/SwapScreen.tsx

```diff
@@ -46,15 +46,23 @@ export function SwapScreen() {
   const { t } = useTranslation()
   const dispatch = useDispatch()
 
-  const coreTokens = useSelector(coreTokensSelector)
+  const supportedTokens = useSelector(coreTokensSelector)
```

Comment:
This will change when we have the cloud function correct?

Reply:
I'm not sure what cloud function you are referring to, but I think we will need to update the selector to return only the assets that we support for swapping.
https://linear.app/valora/issue/RET-611/[wallet]-more-tokens-support